locales/en/messages.json

```diff
@@ -4151,6 +4151,9 @@
     "osdSetupTitle": {
         "message": "OSD"
     },
+    "osdSetupNoOsdChipConfigWarning": {
+        "message": "<span class=\"message-negative\"><b>WARNING:</b></span> No OSD chip was configured. Please, make sure you have <strong>osd_displayport_device</strong> set to <strong>AUTO</strong> mode or correct value for your board using $t(tabCLI.message) tab."
```

Comment:
I'm not good at English at all, so I don't know if it must have article or not:
> set to AUTO mode or **_the_** correct value for your

So I approve it, if not I suppose someone more will complain :P

Reply:
No OSD chip is configured. Please make sure you put the OSD displayport device in AUTO mode or use the correct value for your board using the CLI tab